lib/system.ml

```diff
@@ -225,6 +225,21 @@ let is_in_system_path filename =
     warn_path_not_found ();
     false
 
+let warn_using_current_directory =
+  CWarnings.create ~name:"default-output-directory" ~category:CWarnings.CoreCategories.filesystem
+    (fun s ->
+       Pp.(strbrk "Output directory is unset, using \"" ++ str s ++ str "\"." ++ spc () ++
+           strbrk "Use command line option \"-output-directory to set a default directory."))
```

Comment:
usualy I do
```suggestion
    Pp.(fun s ->
       strbrk "Output directory is unset, using \"" ++ str s ++ str "\"." ++ spc () ++
       strbrk "Use command line option \"-output-directory to set a default directory.")
```
less parentheses that way
(I didn't check indentation in the `` ```suggestion ``)

Reply:
Ok, done.